app/models/container.rb

```diff
@@ -95,6 +96,14 @@ def duplicate
     )
   end
 
+  def update_bootstrappers(bootstrappers)
```

Comment:
test for this portion of the code? @fadlinurhasan

Reply:
Updated Mas.